.gitignore

```diff
@@ -71,9 +71,11 @@ typings/
 # dotenv environment variables file
 .env
 .env.*
+config.json
```

Comment:
@taylorjdawson and I chatted a bit about this — we thought we should move on from storing env vars in a `config.json` and instead inject these variables from `.env` directly

Reply:
But that's a later discussion, probably introduced with Taylor's upcoming Nextjs 14 example

Reply:
Nice! I agree, `.env` is more standard. When we support this we'll have a place to update (this example!). I imagine we'll have a convenience function "turnkey.fromEnv" or similar?